### PR TITLE
Update n.whatwg.org namespace documents

### LINKED ISSUE #99: Add a page at n.whatwg.org/work
It should link to https://html.spec.whatwg.org/#licensing-works (or the multipage variant) and should clarify that despite web browsers redirecting to https://, for the purposes of the namespace you must use http://.

### PR DESCRIPTION
**Pull request by annevk**
Fixes #99.

### CODE REVIEW

**Comment by annevk** [comment]
This requires a corresponding fix on the server to remove the redirect.

cc @foolip @sideshowbarker

File: n.whatwg.org/work [hunk 1]
```diff
@@ -0,0 +1,6 @@
+The string http://n.whatwg.org/work (note the lack of HTTPS compared to
+the URL of this document) represents the namespace for a work. See
+
+  https://html.spec.whatwg.org/multipage/microdata.html#licensing-works
```
Comment (foolip) [inline n.whatwg.org/work]: Is the indentation intentional, given that it was removed in the other file?

Comment (foolip) [reply n.whatwg.org/work]: Ah, it's now two spaces in both files.

**Comment by domenic** [comment]
FWIW when I field #99 I think I must have been unaware of the redirect... The redirect is not too bad IMO. But if by filing #99 I've convinced others that a dedicated pointer page is better, that works too :)

**Comment by foolip** [comment]
I'm fine either way, perhaps with a weak preference for what Anne has done here.

**Pull request merged**